__tests__/links.test.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+const pool = require('../lib/utils/pool');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('Link endpoint', () => {
+  beforeEach(() => {
+    return pool.query(fs.readFileSync('./sql/setup.sql', 'utf-8'));
+  });
+  beforeEach(() => {
+    return pool.query(fs.readFileSync('./sql/test.sql', 'utf-8'));
+  });
+  afterAll(() => {
+    return pool.end();
+  });
+
+  it('creates a new link via POST', async() => {
```

Comment:
Looks good. I'll update my it statements to mimic yours to make them consistent.